docs/content.rst

```diff
@@ -614,6 +614,14 @@ the ``DEFAULT_METADATA``::
 To publish a post when the default status is ``draft``, update the post's
 metadata to include ``Status: published``.
 
+Hidden posts
+=================
+
+Like pages, posts can also be marked as ``hidden`` with the ``Status: hidden``
+attribute. Hidden posts will be output to the final path as expected, but
```

Comment:
> Hidden posts will be **outputted** ...

Is it also worth mentioning that these follow `ARTICLE_SAVE_AS` as opposed to `DRAFT_SAVE_AS`?